packages/stepper/src/components/Step.js

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import PropTypes from "prop-types";
+
+import Container from "../styledComponents/Step";
+import statusTypes from "../constants/statusTypes";
+
+export default class Step extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.onClick = this.onClick.bind(this);
+  }
+
+  onClick(event) {
+    event.stopPropagation();
+
+    if (this.props.item.status !== statusTypes.CURRENT) {
+      return;
+    }
+
+    return this.props.onClick(this.props.item.id);
+  }
+
+  render() {
+    const isCompleted = this.props.item.status === statusTypes.COMPLETED;
+
+    return (
+      <Container status={this.props.item.status} onClick={this.onClick}>
+        {this._renderIcon(isCompleted ? "wg-check" : "wg-checker")}
+
+        <div className="description">{this.props.item.description}</div>
+
+        {this.props.item.status !== statusTypes.COMPLETED &&
+          this._renderIcon("wg-small-arrow-right")}
+      </Container>
+    );
+  }
+
+  _renderIcon(iconName) {
+    return (
+      <div className="icon">
+        <i className={iconName} />
+      </div>
+    );
+  }
+
+  static propTypes = {
```

Comment:
What do you think about putting the `propTypes` on the beginning of the file to make a faster reading of the components properties?

Reply:
Hmm... On our webapp we always placed the `propTypes` at the end of the file.
I know it's not the same project, but i kinda like this way because when I open the file I can get faster to the `render` method which I believe is the most viewed piece of the code :stuck_out_tongue: 
Do you see any other advantages?

Reply:
For example, I developing another component and just wanna check the props of the Step component, it's way faster just to open the file and look at the first lanes then running a storybook or scrolling down to the render method.